buffer/src/main/java/io/netty/buffer/SizeClasses.java

```diff
@@ -78,14 +78,13 @@
  * <p>
  *   ( 76,    24,       22,        1,       yes,            no,        no)
  */
-abstract class SizeClasses implements SizeClassesMetric {
+class SizeClasses implements SizeClassesMetric {
```

Comment:
Consider making it `final` to ensure we lock out inheritance.

Reply:
Done